tests/test_subscription_manager.py

```diff
@@ -17,7 +17,7 @@ def test_lambda_handler(tables):
         tables.subscriptions_table.put_item(Item=item)
 
     with patch('subscription_manager.invoke_worker') as mock_invoke_worker, \
-            patch.dict(os.environ, {'SUBSCRIPTION_WORKER_ARN': TEST_WORKER_ARN}):
+            patch.dict(os.environ, {'SUBSCRIPTION_WORKER_ARN': TEST_WORKER_ARN}, clear=True):
```

Comment:
`clear=True` breaks the test, since it discards the required `SUBSCRIPTIONS_TABLE_NAME` variable that's currently set in `cfg.env`. I think I like setting env vars for testing in the tests themselves, but that risks introducing these kinds of conflicts now that we're setting env vars in multiple places.

Dropping `clear=True` lets the test pass for now, if we want to do that to move forward, but should we use this opportunity to move some values out of cfg.env?

Reply:
Oh that's probably why I didn't include it originally, let's just drop it for now.